index.js

```diff
@@ -202,6 +206,10 @@ function firewall (remotePublicKey) {
     return true
   }
 
+  if (b4a.equals(list.userData.swarmPublicKey, remotePublicKey)) {
```

Comment:
Can we do for safety like `swarmPublicKey && b4a.equals(..)`? Otherwise it throws if the metadata isn't set yet due null